### PR TITLE
Update add-device-images.md

### PR DESCRIPTION
**Pull request by kmsft**
there is a 20-machine simultaneous limit for "managed images" https://learn.microsoft.com/en-us/azure/virtual-machines/windows/capture-image-resource which customers are not subject to because W365 uses Azure Compute Gallery

will reach out to W365 PM about this change

### CODE REVIEW

File: windows-365/enterprise/add-device-images.md [hunk 1]
```diff
@@ -36,6 +36,7 @@ If you want to use a custom device image, you can add it into your Azure subscri
 >
 > - Windows 10 images should not contain Azure Virtual Desktop client components. Additionally, you can't import Windows 10 Multisession images into Windows 365.
 > - For information about support for Windows 11 custom device images, see [What's New for Windows 365 Enterprise](whats-new.md#support-for-windows-11).
+> - Custom device images are uploaded to the Windows 365 [Azure Compute Gallery](https://learn.microsoft.com/en-us/azure/virtual-machines/azure-compute-gallery) which supports over 20 concurrent deployments
```
Comment (JohanFreelancer9) [inline windows-365/enterprise/add-device-images.md:39]:
```suggestion
> - Custom device images are uploaded to the Windows 365 [Azure Compute Gallery](https://learn.microsoft.com/azure/virtual-machines/azure-compute-gallery) which supports over 20 concurrent deployments.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @kmsft, kindly commit to the suggested change. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by ErikjeMS** [comment]
#sign-off

**Comment by elaineyou** [comment]
Azure Computer Gallery is too deep a detail that customers don't need to know. Why don't we use the existing statement in "Device Images" doc as below:

If none of the default gallery images meet your requirements, you can upload up to 20 of your own custom device images.

**Comment by kmsft** [comment]
I will propose: "If none of the default gallery images meet your requirements, you can upload your own custom device image."

The "20" is not a limit on the number of custom images that can be uploaded.  20 is the limit of simultaneous VMs that can be deployed with a single custom image.

The issue is a customer may be lead to believe W365 deployments will be limited to 20 simultaneous CPC deployment if they use a custom image and use the Managed Image method instead of Azure Compute Gallery.

A custom image uploaded to W365 is not subject to the 20 simultaneous deployments, because W365 uses Azure Compute Gallery behind the scenes.

**Comment by ErikjeMS** [comment]
@kmsft Please recreate this PR in the private repo. Closing.

**Pull request closed without merging**